packages/api/src/utils/billing-types.ts

```diff
@@ -58,6 +59,23 @@ export interface CustomersService {
 
 export type StoragePriceName = 'free' | 'lite' | 'pro'
 
+export interface PriceTier {
+  flatAmount: number | null;
+  unitAmount: number | null;
+  upTo: number | null;
+}
+
+export interface CustomStoragePrice {
+  id: string;
+  bandwidth?: string;
```

Comment:
Will you add a comment explaining what this means? I'm a little surprised the bandwidth is expressed as a string vs something like a number amount + (explicit or implicit) a unit involving bytes and time.

e.g.
```
bandwidth: {
  monthlyLimit: { bytes: number }
}
```

Reply:
since we're mapping this, I agree that it does make sense as a number now, and we may use this bandwidth later to control more than just the display (supporting what you were saying before re: business logic etc)